### PR TITLE
Add staff signoff to request deletion

### PR DESCRIPTION
**Pull request by antoinegag**
Require a member of staff with the Kick permissions, so moderator or admin, to mark a request as bad before it's deleted.

### CODE REVIEW

**Review by jriwanek — APPROVED** [review]

File: src/main/java/com/mcmoddev/mmdbot/modules/logging/misc/EventReactionAdded.java [hunk 1]
```diff
@@ -70,15 +72,37 @@ public void onMessageReactionAdd(final MessageReactionAddEvent event) {
...
+                    LOGGER.info(REQUESTS, "Request from {} has a score of {}, reaching removal threshold {}, "
+                            + "awaiting moderation approval.",
+                        messageAuthor, requestScore, removalThreshold);
+
+                    final var logChannel = guild.getTextChannelById(getConfig()
+                        .getChannel("events.requests_deletion"));
+                    if (logChannel != null) {
+                        logChannel.sendMessage(String.format("Request from %s (%s;%s) reached deletion threshold "
+                                + "awaiting moderator approval for deletion", messageAuthor.getAsMention(),
+                            messageAuthor.getAsTag(), messageAuthor.getId(), message.getContentRaw()))
```
Comment (sciwhiz12) [inline src/main/java/com/mcmoddev/mmdbot/modules/logging/misc/EventReactionAdded.java:99]: Two things:

1. Add a `,` after the word `threshold`; and
1. Remove the additional `message.getContentRaw()` argument, as there's no corresponding parameter in the format string, or alternatively add a parameter to the format string such as `%.20s` and pass in `message` directly to include a snippet of the message's contents (as per the [javadocs on Message](https://ci.dv8tion.net/job/JDA/javadoc/net/dv8tion/jda/api/entities/Message.html)).

**Review by sciwhiz12 — APPROVED** [review]
Looks good to me, aside from two small changes.

**Review by williambl — APPROVED** [review]

**Comment by antoinegag** [comment]
Proxy recommended that we foward these requests to a moderator-only channel to be signed off there, using the the new button API. 

Not going to do it in this PR but leaving that here to be done later.

File: src/main/java/com/mcmoddev/mmdbot/modules/logging/misc/EventReactionAdded.java (hunk 1 above)
Comment (antoinegag) [reply src/main/java/com/mcmoddev/mmdbot/modules/logging/misc/EventReactionAdded.java:99]: Oops, copy paste error, good catch!

**Pull request merged**